go.mod

```diff
@@ -67,6 +67,12 @@ require (
 	pgregory.net/rapid v1.2.0
 )
 
+require (
+	github.com/go-logr/logr v1.3.0 // indirect
```

Comment:
Hmm, I wonder why this is added as a separate section?

Reply:
Hmm, yeah this is indeed weird.. Joined it with the previous section.